components/player/VideoPlayer.tsx

```diff
@@ -1,15 +1,57 @@
-import React from 'react';
-import { View } from 'react-native';
+/* eslint-disable react/no-unstable-nested-components */
+import { useTheme } from '@react-navigation/native';
+import { ResizeMode, Video } from 'expo-av';
+import { Image } from 'expo-image';
+import React, { useRef } from 'react';
+import { Dimensions, StyleSheet, View } from 'react-native';
+import { VIDEO_URL } from '../../config/constants';
 
-// TODO: Add VideoPlayer component
-/**
- * -  Implement a video player card component
- * - Hint: use `expo-av` to implement the video player with hidden controls
- * - consider adding fullscreen player
- */
+const { width } = Dimensions.get('window');
 
 const VideoPlayer = () => {
-  return <View />;
+  const { colors } = useTheme();
+
+  const video = useRef(null);
+
+  return (
+    <View style={[styles.container, { backgroundColor: colors.card }]} testID="video-player">
+      <Video
+        testID="video-component"
+        ref={video}
+        style={styles.video}
+        source={{
+          uri: VIDEO_URL,
+        }}
+        useNativeControls
+        resizeMode={ResizeMode.CONTAIN}
```

Comment:
We are currently using `resizeMode={ResizeMode.CONTAIN}`, notice the edges of the container are not filled by the video. We should maybe use `COVER` and `CONTAIN` only in full screen mode.

![image](https://github.com/droidconKE/droidconKE2023ReactNative/assets/5578688/1a31e0bd-4797-4c59-b1ad-cf00a4f361d1)

Reply:
actually `ResizeMode.CONTAIN` fixes this and doesn't scale the video so much